utils/genesis/src/lib.rs

```diff
@@ -161,6 +162,10 @@ where
             .tipset_by_height(0, ts.clone(), true)
             .await?;
         sm.chain_store().set_genesis(&gb.blocks()[0])?;
+        if !matches!(&sm.chain_config().genesis_cid, Some(expected_cid) if expected_cid ==  &gb.blocks()[0].cid().to_string())
+        {
+            bail!("Snapshot incompatible with {}. Consider specifying the network with `--chain` flag or use a custom config file to set expected genesis CID for selected network", sm.chain_config().name);
+        }
```

Comment:
Could you wrap these lines to 100 or 120 characters?

Reply:
done